_rules/SC1-1-1-accessible-name-is-image-filename.md

```diff
@@ -0,0 +1,89 @@
+---
+name: Image accessible name is filename
+description: |
+  This rule checks that image elements that use their source filename as their accessible name do so without loss of infomation to the user.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to HTML `input` element with a [`type`](https://www.w3.org/TR/html/sec-forms.html#dom-htmlinputelement-type) of `image`, or `area` elements, or any HTML element with the semantic role of `img`, where the element is [included in the accessibility tree](#included-in-the-accessibility-tree) and has an [accessible name](#accessible-name) that is equivalent to the [filename](#filename) specified in the `src` attribute.
+
+**Note**: Difference in letter casing, and forward and trailing whitespace should be ignored. 
```

Comment:
As it's suggested in the #251 ([see comment](https://github.com/auto-wcag/auto-wcag/pull/251#discussion_r229372629)), this tests should only be applicable to non decorative image ?

Reply:
I should have detailled more my though, sorry  :)

In fact, the image that would be tested can have an alt, and so be included in accessibility tree. But, imagine that image is, in the end, a decorative image. That test should not apply, because that image should be evaluate regarding a test that determine if it has a null alt (or whatever that makes it decorative). But the validity of the text of the alt should not be tested.

Reply:
updated applicability based on @audreymaniez comments in agreement with @jkodu and @annethyme .
Also updated the [decorative](https://github.com/auto-wcag/auto-wcag/blob/Brynanders-patch-1/pages/algorithms/decorative.md) definition to include a note taken from **Image has name** #274 on how to mark an image as 'decorative'. 
Also made a new commit for [**Image has name** here](https://github.com/auto-wcag/auto-wcag/blob/Brynanders-patch-1/_rules/SC1-1-1-image-has-name.md).